ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/SummaryIntegrationTests.cs

```diff
@@ -199,6 +199,31 @@ public async Task Should_show_the_selected_decline_reasons_and_details()
 			declineReasonSummary.Should().NotContain("Other", because: "Other was not selected");
 		}
 
+		[Fact]
+		public async Task Should_show_the_selected_deferred_reasons_and_details()
+		{
+			AcademyConversionProject project = AddGetProject(p => p.GeneralInformationSectionComplete = false);
+
+			var wizard = new RecordDecisionWizard(Context);
+
+			await wizard.StartFor(project.Id);
+			await wizard.SetDecisionToAndContinue(AdvisoryBoardDecisions.Deferred);
+			await wizard.SetDecisionByAndContinue(DecisionMadeBy.Minister);
+			await wizard.SetDeferredReasonsAndContinue(
+				Tuple.Create(AdvisoryBoardDeferredReason.PerformanceConcerns, "Performance detail"),
+				Tuple.Create(AdvisoryBoardDeferredReason.Other, "Other"),
+				Tuple.Create(AdvisoryBoardDeferredReason.AdditionalInformationNeeded, "additional info"),
+				Tuple.Create(AdvisoryBoardDeferredReason.AwaitingNextOftedReport, "Ofsted"));
+			await wizard.SetDecisionDateAndContinue(DateTime.Today);
+
+			string declineReasonSummary = Document.QuerySelector("#deferred-reasons").TextContent;
+
+			declineReasonSummary.Should().Contain("Additional information needed:");
```

Comment:
Should also check that the explanation text is present.

Reply:
Have addressed this.